flake.nix

```diff
@@ -0,0 +1,36 @@
+{
+  description = "CMK Config Broker";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        lib = import <nixpkgs/lib>;
+        pkgs = nixpkgs.legacyPackages.${system};
+      in
+      rec {
+        packages = {
+          cmk-config-broker = {
+            name = "cmk-config-broker";
```

Comment:
change `cmk-config-broker` to `recrypt-node-binding` all over. You may want to remove the package alltogether and just provide a dev shell, since I think building this will be more complicated.